package.json

```diff
@@ -78,6 +77,7 @@
     "lint-staged": "^13.1.0",
     "lodash.camelcase": "^4.3.0",
     "prettier": "^3.2.5",
+    "rollup": "^4.14.1",
```

Comment:
not sure why these were pinned, but lgtm, pending a quick SDK playground smoke-test

Reply:
did a smoke test, appears to be good :)

Reply:
thought waiting for an actual stamp